README.md

```diff
@@ -7,70 +7,65 @@ debt in the JS ecosystem.
 Our goal is to provide a cleaner, faster and simpler dependency tree in the JS
 ecosystem for those who are concerned with it.
 
-## The problem
+## How to help
 
-Packages often have dependencies, and those dependencies themselves have
-dependencies, and so on. This is fine and expected with how dependency trees
-are structured.
+All issues on our issue tracker are labeled and a lot of them contain actionable items you can start to work on right away.
+Hence, an explanation of the labels is in order:
```

Comment:
I wouldn't add an explanation for all the labels as it's likely to get out-of-sync as we add, remove, update labels. The labels already have descriptions that are shown on GitHub when the label is hovered.

I think it'd be okay to call out one or two labels here like "accepts prs" that might help folks get started more quickly